proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -147,6 +147,11 @@ message DispatchDetail {
 
   // UTC Timestamp of the last update to the order.
   google.protobuf.Timestamp modification_time = 4;
+
+  // UTC Timestamp when the dispatch will stop working.
+  // Will be calculated internally based on the given: start_time,
+  // duration and RecurenceRule
+  google.protobuf.Timestamp estimated_end_time = 5;
```

Comment:
I wonder if we should just call it "end_time", it's not really an education but a precise calculation, no?

Reply:
Good point, done :)